### ISSUE TITLE
Provide binaries for releases 

### ISSUE THREAD

**Issue by Asone** [scsibug/nostr-rs-relay#115]
Hi, 

I'm currently trying to install nostr-rs-relay. However i'm having a hard time with both options described in the `README`, be it with or without Docker. 

I don't fear too much as i'm used to handled such difficulties so i'll get through it. 

However i am a bit surprised that no binaries are provided for the releases to allow people to just download an go with the service, which helps people with few rust knowledge to avoid headaches regarding compilation. 

Could it be a possible option to provide some binaries when shipping releases ? This is what i did for [nostrss](https://github.com/Asone/nostrss). You can find the Github actions to ship the binaries as assets on release on this [Github action file](https://github.com/Asone/nostrss/blob/main/.github/workflows/release.yml)

Many thanks

**Comment by scsibug** [comment]
Good idea, but this is going to be a little bit more difficult than just pulling in your excellent example, I think.  It did not seem like the cross docker images had a recent enough version of protobuf-compiler for our setup here.  I will keep playing with it, but it seems like we either need a new base image, or to not use cross and limit the release to a single platform.

**Comment by scsibug** [comment]
Builds are available from sourcehut; example: https://builds.sr.ht/~gheartsfield/job/1105920

Still planning on producing binaries for releases, I'll try to do that for the next one.  Not 100% sure where to host it yet.  The sourcehut binaries are pruned after 90 days, so that won't work; but perhaps I can attach to the tagged release over there.
